src/main/resources/Datavyu_API.rb

```diff
@@ -432,15 +433,14 @@ def sort_cells()
   def change_code_name(old_name, new_name)
     i = @old_args.index(old_name)
     @old_args[i] = new_name
-    if ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"].include?(old_name[1].chr)
```

Comment:
Does the RColumn.sanitize.codename function will check this condition and process the arg accordingly, or it is obsolete and could be removed?